firefox_puppeteer/ui/windows.py

```diff
@@ -422,18 +426,16 @@ def is_private(self):
 
     @use_class_as_property('ui.toolbars.NavBar')
     def navbar(self):
-        """
-        Provides access to the navigation bar. This is the toolbar containing
+        """Provides access to the navigation bar. This is the toolbar containing
         the back, forward and home buttons. It also contains the location bar.
 
-        See the :class:`~ui.navbar.NavBar` reference.
+        See the :class:`~ui.toolbars.NavBar` reference.
         """
 
     @property
     def tabbar(self):
-        """
-        Provides access to the tab bar. This is the toolbar containing all the
-        tabs, the new tab button, and the tab menu
+        """Provides access to the tab bar. This is the toolbar containing all the
+        tabs, the new tab button, and the tab menu.
```

Comment:
The description should fit into the first line. I don't think that we need the second sentence, so lets remove it.